docs/develop/01_sdk/02_Cookbook/01_Manage a KILT DID/03_full_did_update.md

```diff
@@ -0,0 +1,13 @@
+---
+id: full-did-update
+title: Update a full DID
+---
+
+import CodeBlock from '@theme/CodeBlock';
+import Example5 from '!!raw-loader!@site/code-examples/core_features/did/5_did.ts';
+
+Once anchored on the KILT blockchain, a full DID can be updated. For instance, the following snippet shows how to use the `FullDidUpdateBuilder` to update the authentication key to a new `ed25519` key and remove a service endpoint for a full DID.
```

Comment:
M: Incorrect preposition and new line please.
```suggestion
Once anchored to the KILT blockchain, a full DID can be updated.
For instance, the following snippet shows how to use the `FullDidUpdateBuilder` to update the authentication key to a new `ed25519` key and remove a service endpoint for a full DID.
```

Reply:
https://github.com/KILTprotocol/docs/pull/101/commits/2d56013ffea296ca7278656c11f6b6b6b1430ed6 and https://github.com/KILTprotocol/docs/pull/101/commits/401e87b0810efdae4ed4a7668e3bd1e5444c9553